docs/command-line-interface/aserto-idp-cli/configuration.mdx

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_label: Configuration
+---
+
+# Configuration
+
+To configure the CLI plugins, create a YAML file with the following contents (including the information for the plugins you intend to use):
+
+```
+logging:
+  log_level: LEVEL
+plugins:
+  auth0:
+    domain: DOMAIN
+    client-id: ID
+    client-secret: SECRET
+  json:
+    from-file: PATH_TO_FILE
+    to-file: PATH_TO_OUTPUT_FILE
+  aserto:
+    tenant: TENANT
+    authorizer: AUTHORIZER
+    api-key: API_KEY
+  okta:
+    domain: OKTA_DOMAIN
+    api-token: TOKEN
+```
+
+:::note
```

Comment:
seems like instead of this one note, we should have a section for each provider type (not necessarily a whole page - just a section).